canonical-web-and-design/vanilla-framework
# Extract accessibility docs

## Done

- Extract the accessibility docs for the components which have them
- Create a folder for each component, include `index.md` and `accessibility.md` for each component. This keeps the url structure in a logical order e.g. `/docs/patterns/accordion/accessibility`

Fixes [#1311](https://github.com/canonical-web-and-design/vanilla-squad/issues/1311)

## QA

- Open [demo](insert-demo-url)
- Visit the following pages to check there are no 404s:
  - https://vanilla-framework-4413.demos.haus/docs/patterns/accordion and https://vanilla-framework-4413.demos.haus/docs/patterns/accordion/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/breadcrumbs and https://vanilla-framework-4413.demos.haus/docs/patterns/breadcrumbs/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/buttons and https://vanilla-framework-4413.demos.haus/docs/patterns/buttons/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/card and https://vanilla-framework-4413.demos.haus/docs/patterns/card/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/contextual-menu and https://vanilla-framework-4413.demos.haus/docs/patterns/contextual-menu/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/grid and https://vanilla-framework-4413.demos.haus/docs/patterns/grid/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/heading-icon and https://vanilla-framework-4413.demos.haus/docs/patterns/heading-icon/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/icons and https://vanilla-framework-4413.demos.haus/docs/patterns/icons/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/links and https://vanilla-framework-4413.demos.haus/docs/patterns/links/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/lists and https://vanilla-framework-4413.demos.haus/docs/patterns/lists/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/logo-section and https://vanilla-framework-4413.demos.haus/docs/patterns/logo-section/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/modal and https://vanilla-framework-4413.demos.haus/docs/patterns/modal/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/navigation and https://vanilla-framework-4413.demos.haus/docs/patterns/navigation/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/notification and https://vanilla-framework-4413.demos.haus/docs/patterns/notification/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/search-box and https://vanilla-framework-4413.demos.haus/docs/patterns/search-box/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/status-labels and https://vanilla-framework-4413.demos.haus/docs/patterns/status-labels/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/strip and https://vanilla-framework-4413.demos.haus/docs/patterns/strip/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/switch and https://vanilla-framework-4413.demos.haus/docs/patterns/switch/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/tabs and https://vanilla-framework-4413.demos.haus/docs/patterns/tabs/accessibility
  - https://vanilla-framework-4413.demos.haus/docs/patterns/tooltips and https://vanilla-framework-4413.demos.haus/docs/patterns/tooltips/accessibility

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
